test/test.c

```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
```

Comment:
Windows does not have unistd.h unfortunately

Reply:
will fix thx.